### ISSUE TITLE
about the fusion code

### ISSUE THREAD

**Issue by CanCanZeng** [GhiXu/ACMP#7]
Hi GhiXu, thank you for your great work!
I tested the code and it really produce beautiful result! And Since I'm more familiar with colmap, so I modified your code to read colmap project directly, so I do not have to use the script to convert project type. I opened the project in https://github.com/CanCanZeng/ACMP_colmap, but I can delete it if you do not like it.

I looked into the depth fusion code, it's very similar to the original fusion code in colmap, and I set the parameters to the same, but the fused point cloud is much different, I can't find the cause. Have you find this situation?

The images I used is from https://github.com/openMVG/ImageDataset_SceauxCastle, and the SfM is done by colmap.
the depth map is produced by ACMP on original resolution. the point cloud fused by ACMP and by colmap is much different
![Screenshot from 2021-06-16 21-50-52](https://user-images.githubusercontent.com/26817559/122231571-1c29bb00-ceed-11eb-9835-0f1b6ff65641.png)
![Screenshot from 2021-06-16 21-51-08](https://user-images.githubusercontent.com/26817559/122231589-21870580-ceed-11eb-91fb-ae48dc12d3ed.png)
the later is result from ACMP.
I think the key parameters are reproj_error, relative_depth_diff, angle and num_consistent, I set them equal to colmap defaul value. I do not mean that the point cloud produced by colmap is better, I just want to know why they are so different.

**Comment by GhiXu** [comment]
The fusion in colmap defines a directed graph of consistent pixels. This makes the default value of num_consistent larger than the one in my fusion code.

**Comment by CanCanZeng** [comment]
Yeah, I know this difference, so I set num_consistent >= 4 here https://github.com/GhiXu/ACMP/blob/604e2347f8a7d302ebaa4cb619b0bf2e87527853/main.cpp#L285
